src/app/wallet/wallet.component.html

```diff
@@ -145,7 +145,7 @@ <h4 class="init-left__title">Your private, secure wallet</h4>
                                                       *ngIf="transaction.state === transactionState.MINED">
                                                     <span class="state__invested"
                                                           *ngIf="transaction.type === transactionType.INVEST">
-                                                        {{ transaction.share | percent: "1.0-10" }} Invested —
+                                                        {{ transaction.share | percent: '1.0-10' }} Invested —
```

Comment:
```suggestion
                                                        {{ transaction.share | percent:'1.0-10' }} Invested —
```

Reply:
This was fixed in the last commit (https://github.com/AMPnet/ampnet-crowdfunding-frontend/pull/152/commits/df33199816a7d0a612d26a23335a8005518b6a2c)